client/app/common/translations/Messages_fr_FR.json

```diff
@@ -551,6 +551,7 @@
   "common_database_version_postgresql_95": "PostgreSQL 9.5",
   "common_database_version_postgresql_96": "PostgreSQL 9.6",
   "common_database_version_postgresql_10": "PostgreSQL 10",
+  "common_database_version_postgresql_11": "PostgreSQL 11",
```

Comment:
We should find a way to automatize this

Reply:
And this should also be a constant considering the text does not change over languages ?

Reply:
It could be handle as the same way that it was done here:

- ovh-ux/ovh-manager-web#1141 (reported JIRA: MANAGER-2973)

Since this ticket has been validated by QC I will merge it.

Feel free to open a new ticket/pull request related to this change.

Thanks!

Reply:
It could be handle as the same way that it was done here:

- ovh-ux/ovh-manager-web#1141 (reported JIRA: MANAGER-2973)

Since this ticket has been validated by QC I will merge it.

Feel free to open a new ticket/pull request related to this change.

Thanks!